web-ui/src/pages/OnboardProgressPage.jsx

```diff
@@ -161,7 +193,10 @@ export default function OnboardProgressPage() {
                   disablePortal
                   options={posOptions}
                   sx={{ width: "75%" }}
-                  renderInput={(option) => <TextField variant="outlined" {...option} />} />
+                  renderInput={(option) => (
+                    <TextField variant="outlined" {...option} />
+                  )}
+                />
               </Grid>
               <Grid item xs={6}>
                 <Typography id="description" sx={{ mt: 2 }}>
```

Comment:
Typography needs to be closed </Typography>

Reply:
Its closed already